chapters/modules/chapter_1/pages/why_starknet.adoc

```diff
@@ -0,0 +1,21 @@
+[id="whyStarknet"]
+
+= Why StarkNet?
+
+== The Problem
+Ethereum scaling has been a major discussion topic since the network was launched. Commonly known as the blockchain trilemma, most applications have to choose 2/3 features between scalability, security and decentralization and compromise on any one of them.
+
+Several NFT launches have seen the gas fee spike up to 1000s of dollars congesting the network, thereby casting doubts on Ethereum's ability to scale.
+
+== The Solution
+
+Layer 2 solutions can offer scalability to Ethereum by helping off-load a lot of computational work done at the Ethereum layer to another layer. Roll up solutions have the potantial to scale Ethereum in a way that billions of users across the globe can use it seamlessly.
```

Comment:
We are not providing more computing power to Ethereum itself, we are just using Ethereum's computing power for security. Only a zk-EVM type 1 could do what you are describing and that doesn't exist yet

Reply:
If we are taking about scalability - aren't rollup solutions meant to create more computing power for Ethereum? Maybe not directly but indirectly (and that's why they are L2). Ethereum still has to do computational work but not as much as it would be doing without L2. That's what I wanted to convey.
Wdyt?